apps/testingaccessibility/src/utils/algolia/sanity-algolia-client.ts

```diff
@@ -1,19 +1,22 @@
-import algoliasearch from 'algoliasearch'
-import indexer, {flattenBlocks} from 'sanity-algolia'
+import algoliaSearchClientCreator from 'algoliasearch'
+import sanityAlgoliaIndexer from 'sanity-algolia'
```

Comment:
I did some renaming in here because all the "clients" and "apis" were confusing to me 😅